assets/components/src/plugin-settings/SettingsSection.js

```diff
@@ -100,7 +110,7 @@ const SettingsSection = props => {
 					'buttons',
 					<Button
 						variant="primary"
-						disabled={ disabled }
+						disabled={ sectionSettingsButtonDisabled }
```

Comment:
I think this should be `disabled || sectionSettingsButtonDisabled`. What happens now is that when you click save the button is still active while processing the request

Reply:
Hi @leogermani, The above feedback is being addressed. Thanks.